diff --git a/dist/plugin.js b/dist/plugin.js
--- a/dist/plugin.js
+++ b/dist/plugin.js
@@ -142,7 +142,7 @@
                 allureVersion: context.allureVersion,
                 reportFiles: context.reportFiles,
                 reportUuid: context.reportUuid,
-                reportName: context.reportName,
+                reportName: this.options.reportName || context.reportName,
                 ci: context.ci,
                 executor,
                 runSummary,
